ios/Litentry.xcodeproj/project.pbxproj

```diff
@@ -20,6 +20,7 @@
 		2D02E4BF1E0B4AB3006451C7 /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB71A68108700A75B9A /* main.m */; };
 		2DCD954D1E0B4F2C00145EB5 /* LitentryTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 00E356F21AD99517003FC87E /* LitentryTests.m */; };
 		32474834DF705BF2BD74A5CB /* libPods-Litentry.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 86CB3E182E033C48FE853073 /* libPods-Litentry.a */; };
+		46E75EBE2897CB1100C6286B /* macos in Resources */ = {isa = PBXBuildFile; fileRef = 46E75EBD2897CB1100C6286B /* macos */; };
```

Comment:
All changes in this file are related to linking macos, it should be reverted.. This is not a file to edit manually unless you know what you are doing. always use xcode

Reply:
Is this removed?

Reply:
Any forwarding links? I'm not able to make any progress on this.

Reply:
How this change was introduced? can you recall?

Reply:
I just followed the instructions for installing the package.
https://github.com/react-native-netinfo/react-native-netinfo#using-react-native--060

Reply:
I see, then this changes should be automatic. We can keep it then.